src/char/sorceress.py

```diff
@@ -19,6 +20,23 @@ def __init__(self, skill_hotkeys, char_config, screen: Screen, template_finder:
         super().__init__(skill_hotkeys, char_config, screen, template_finder, ui_manager)
         self._pather = pather
 
+    def pick_up_item(self, pos: Tuple[float, float], item_name: str = None, prev_cast_start: float = 0):
+        if self._skill_hotkeys["telekinesis"] and any(x in item_name for x in ['potion', 'misc_gold']):
+            keyboard.send(self._skill_hotkeys["telekinesis"])
+            wait(0.1, 0.2)
+            mouse.move(pos[0], pos[1])
+            wait(0.1, 0.2)
+            mouse.click(button="right")
+            # need about 0.4s delay before next capture for the item not to persist on screen
+            cast_start = time.time()
+            interval = (cast_start - prev_cast_start)
+            cast_duration_wait = (self._cast_duration - interval)
+            delay = 0.35 if cast_duration_wait <0 else (0.35+cast_duration_wait)
+            wait(delay,delay+0.1)
+            return cast_start
+        else:
+            super().pick_up_item(pos)
```

Comment:
return here and pass through params
```suggestion
            return super().pick_up_item(pos, item_name, prev_cast_start)
```